### PR TITLE
fix: add missing frontMatter option to viewer

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* fixed that add missing frontMatter option to viewer

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/src/js/editor.js [hunk 1]
```diff
@@ -202,7 +202,8 @@ class ToastUIEditor {
       extendedAutolinks,
       referenceDefinition,
       customParser: parser,
-      frontMatter
+      frontMatter,
+      customProp: { showFrontMatter: true }
```
Comment (seonim-ryu) [inline apps/editor/src/js/editor.js:206]: 이전에 하나 놓쳤던 것이 있는듯 한데요ㅎ
에디터 및 뷰어 모듈에서 `this.option.frontMatter = false` 초기값을 지정해주는 것이 누락된 것 같습니다.
(API 옵션 주석에서도 누락되었네요 ㅜㅜ)

>  에디터 : https://github.com/nhn/tui.editor/blob/8349ca52afa2222299cd2e8599077e6342db1ac0/apps/editor/src/js/editor.js#L135
> 뷰어 : https://github.com/nhn/tui.editor/blob/8349ca52afa2222299cd2e8599077e6342db1ac0/apps/editor/src/js/viewer.js#L48

그리고 위에 `customProp. showFrontMatter` 값이 `true`로 지정되어 있는데 `frontMatter` 값을 할당받아서 사용해야 하지 않을까요??

```js
// ...
frontMatter,
customProp: { showFrontMatter: frontMatter }
```

File: apps/editor/index.d.ts [hunk 2]
```diff
@@ -133,6 +133,7 @@ declare namespace toastui {
     nodeId: boolean;
     tagFilter: boolean;
     convertors?: CustomHTMLRendererMap;
+    customProp: Record<string, any>;
   }
```
Comment (seonim-ryu) [inline apps/editor/index.d.ts:137]: `ViewerOptions`에 `frontMatter` 추가가 필요합니다ㅎ

> https://github.com/nhn/tui.editor/blob/610cb52ffc8bf7fae8db733dae096e4cfb15b2a6/apps/editor/index.d.ts#L269

**Review by seonim-ryu — APPROVED** [review]
[9/7] 리뷰 완료합니다. 옵션 부분 확인 부탁드릴게요! 고생 많으셨어요~

File: apps/editor/src/js/editor.js (hunk 1 above)
Comment (js87zz) [reply apps/editor/src/js/editor.js:206]: @seonim-ryu 
아 그러네요 수정하겠습니다~!

**Review by shiren — COMMENTED** [review]
수고하셨습니다.

**Pull request merged**